server/src/helpers/following.js

```diff
@@ -79,14 +79,14 @@ class FollowingHelper {
       if (!user) {
         return 'notAUser';
       }
-      const checkFollowing = await user.countFollower({ followerId });
```

Comment:
Nice work @anasey01 and @uwaelpis everything seems good, but am a little lost here?
Is this part of the notification feature?